### PR TITLE
About Modal should provide guidance on build information

### LINKED ISSUE #606: About Modal should provide guidance on build information
This is an enhancement to the existing [About Modal design pattern](http://www.patternfly.org/pattern-library/communication/about-modal/#design)

The call out image should have 6 callouts:
1. Modal
2. Close Button
3. Title ** new
4. Content ** new
5. Trademark & Copyright info ** new
6. Corner Graphic

As an interim design, we are utilizing the following:
When version and build information are both shown:
Version 6.3 (Build 5)
When version only:
Version 6.3
When build only:
Build 5

### PR DESCRIPTION
**Pull request by zhutaoredhat**
Updated the call out image from 3 callouts to 6 callouts：

    1.Modal
    2.Close Button
    3.Title ** new
    4.Content ** new
    5.Trademark & Copyright info ** new
    6.Corner Graphic

Closes #606

### CODE REVIEW

File: pattern-library/communication/about-modal/design/design.md [hunk 1]
```diff
@@ -4,7 +4,16 @@
 
 1. **Modal:** The About Modal leverages the existing [modal](http://www.patternfly.org/pattern-library/widgets/#modal) in PatternFly. Upon opening the modal, the background behind it should “dim” in order to provide a focused view, reducing confusion. The modal should use the default position; top and centered on the screen.
 1. **Close Button:** Clicking the close button (pficon-close) will dismiss the modal and return the background to it’s original state.
-1. **Content:** Title of product, label and version, and legal text are present. Adequate spacing and font weight consideration should be provided for legibility. Two columns are available for versions that contain both a release name and version number or in the event more space is needed.
+1. **Title:** Title of product.
+1. **Content:** Label and version. Adequate spacing and font weight consideration should be provided for legibility. Two columns are available for versions that contain both a release name and version number or in the event more space is needed.<br>
+As an interim design, we are utilizing the following:<br>
```
Comment (mcarrano) [inline pattern-library/communication/about-modal/design/design.md]: Can you remove the text "As an interim design..."

File: pattern-library/communication/about-modal/design/design.md [hunk 2]
```diff
@@ -4,7 +4,16 @@
...
+1. **Title:** Title of product.
+1. **Content:** Label and version. Adequate spacing and font weight consideration should be provided for legibility. Two columns are available for versions that contain both a release name and version number or in the event more space is needed.<br>
+We are utilizing the following:<br>
+When version and build information are both shown:
+Version 6.3 (Build 5)<br>
+When version only:
+Version 6.3<br>
+When build only:
+Build 5
+1. **Trademark & Copyright info:** Legal text are present.
```
Comment (serenamarie125) [inline pattern-library/communication/about-modal/design/design.md]: What if we did something like 

Trademark and Copyright Information: Optionally include legal text.
Suggested format for copyright: Copyright (c) 2018 Company

**Comment by zhutaoredhat** [comment]
@serenamarie125 @mcarrano  That's make sense to me. I have updated it.

**Comment by serenamarie125** [comment]
@mcarrano since you've already reviewed, I will merge.

**Pull request merged**